src/Messages/OWSDisappearingMessagesJob.m

```diff
@@ -18,6 +18,9 @@
 @interface OWSDisappearingMessagesJob ()
 
 // This property should only be accessed on the serialQueue.
+@property (nonatomic, readonly) TSStorageManager *storageManager;
```

Comment:
I've removed these comments but added others indicating which methods should only be called on the `serialQueue`.